jwst/resample/resample.py

```diff
@@ -363,17 +363,30 @@ def update_exposure_times(self, output_model):
         """Modify exposure time metadata in-place"""
         total_exposure_time = 0.
         exposure_times = {'start': [], 'end': []}
+        effinttm, duration = 0.0, 0.0
         for exposure in self.input_models.models_grouped:
-            total_exposure_time += exposure[0].meta.exposure.exposure_time
-            exposure_times['start'].append(exposure[0].meta.exposure.start_time)
-            exposure_times['end'].append(exposure[0].meta.exposure.end_time)
+            for n, _ in enumerate(exposure):
+                total_exposure_time += exposure[n].meta.exposure.exposure_time
+                exposure_times['start'].append(exposure[n].meta.exposure.start_time)
+                exposure_times['end'].append(exposure[n].meta.exposure.end_time)
+                effinttm += exposure[n].meta.exposure.integration_time
```

Comment:
don't include `integration_time`. That's an individual exposure quantity. Only include EFFEXPTM=XPOSURE, and DURATION=TELAPSE.

EFFEXPTM=meta.exposure.exposure_time
XPOSURE=meta.exposure.effective_exposure_time
DURATION=meta.exposure.duration
TELAPSE=meta.exposure.elapsed_exposure_time

So just create sums for `effexptm` and `duration`. Then below ...

Reply:
done